src/main/java/com/minecolonies/coremod/colony/buildings/BuildingBuilder.java

```diff
@@ -39,13 +38,13 @@
     /**
      * Tags to store the needed resourced to nbt.
      */
-    private static final String TAG_RESOURCE_LIST = "resources";
+    private static final String TAG_RESOURCE_LIST = "resourcesItem";
     private static final String TAG_AMOUNT        = "amount";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "TAG_AMOUNT" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1068)